test/functional/apps/dashboard/group3/dashboard_state.ts

```diff
@@ -209,210 +192,108 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.dashboard.waitForRenderComplete();
     };
 
-    describe('Directly modifying url updates dashboard state', () => {
-      before(async () => {
-        await PageObjects.dashboard.gotoDashboardLandingPage();
-        await PageObjects.dashboard.clickNewDashboard();
-        await PageObjects.timePicker.setHistoricalDataRange();
-      });
+    describe('for embeddable config color parameters on a visualization', () => {
```

Comment:
Looks like we aren't reliably in edit mode at this point, so the test is getting stuck when trying to add the pie chart from the library. We started from a fresh dashboard previously, so we should probably do the same here :)